WBooks/TabBar/TabBarController.swift

```diff
@@ -12,15 +12,20 @@ class TabBarController: UITabBarController {
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        let navLibrary = createNavigationItem(controller: LibraryViewController(), tag: 0, title: "TITLE_LIBRARY", imageUnselect: .libraryUnSelect, imageSelect: .librarySelect)
+        let _libraryViewModel = LibraryViewModel()
+        let navLibrary = createNavigationItem(controller: LibraryViewController(libraryViewModel: _libraryViewModel), tag: 0, title: "TITLE_LIBRARY", imageUnselect: .libraryUnSelect, imageSelect: .librarySelect)
```

Comment:
Add break line after each comma to improve the readability. Like
```
let navLibrary = createNavigationItem(controller: LibraryViewController(libraryViewModel: _libraryViewModel),
                                      tag: 0, 
                                      title: "TITLE_LIBRARY", 
                                      imageUnselect: .libraryUnSelect, 
                                      imageSelect: .librarySelect) 
```

Reply:
Also, we can init the controller before this to be more clearer

Reply:
perfect, in a moment I will make the adjustment

Reply:
Done